### PR TITLE
WE-378: prefetching error on page load and hover

### PR DESCRIPTION
**Pull request by EvanYoung-SP**
### What Changed

Updates the netlify-nextjs plugin to latest, fixes a content issue where links were incorrect.

*NOTE: Technically, this is fixed in WE-377 because there was a newer version that came out (4.1.1) that WE-377 updated to. But there are changes in this PR we should still push through.

### How To Test or Verify
- Load the home page and verify the 404 console errors are not appearing on page load or link hover.
  - https://deploy-preview-639--support-docs.netlify.app/docs

### CODE REVIEW

File: content/momentum/4/operations-riak-backups.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+title: 'Backups'
+description: 'Riak data is distributed across the nodes in the cluster Provided that you have enough nodes remaining online the need to restore from a backup should be quite a rare event Riak offers the ability to backup all data present on a given node or all data present in the...'
 ---
 
 Riak data is distributed across the nodes in the cluster. Provided that you have enough nodes remaining online, the need to restore from a backup should be quite a rare event.
 
 Riak offers the ability to backup all data present on a given node or all data present in the cluster. The backup is stored in a file named by the administrator. This backup can later be used to restore the state of the node or cluster depending on the nature of the backup file.
 
-For details about backing up Riak, consult the [Riak](http://http://docs.basho.com/riak/latest/) documentation.
+For details about backing up Riak, consult the [Riak](http://docs.basho.com/riak/latest/) documentation.
```
Comment (maxharrisnet) [inline content/momentum/4/operations-riak-backups.md:11]: HTTP/S

File: content/momentum/4/operations-riak-ports.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
 None of these ports should be exposed to the public Internet; we strongly recommend that access to these ports from untrusted networks be blocked by a firewall.
 
 ### <a name="operations.riak.ports.erlang"></a> Erlang and Firewalls
 
-The maximum number of concurrent ports used by Erlang is defined in the `/opt/msys/3rdParty/riak/etc/vm.args` file by the variable `ERL_MAX_PORTS`. If port usage needs to be restricted, we recommend defining a range of ports equal to the number of nodes in the cluster. For details on network security and firewall configurations, consult the [Riak](http://http://docs.basho.com/riak/latest/) documentation.
\ No newline at end of file
+The maximum number of concurrent ports used by Erlang is defined in the `/opt/msys/3rdParty/riak/etc/vm.args` file by the variable `ERL_MAX_PORTS`. If port usage needs to be restricted, we recommend defining a range of ports equal to the number of nodes in the cluster. For details on network security and firewall configurations, consult the [Riak](http://docs.basho.com/riak/latest/) documentation.
```
Comment (maxharrisnet) [inline content/momentum/4/operations-riak-ports.md:21]: HTTP/S

File: content/momentum/4/riak.md [hunk 3]
```diff
@@ -1,30 +1,29 @@
...
 ---
 
-
-<a name="idp3607504"></a> 
+<a name="idp3607504"></a>
 
 Riak is a distributed key-value data storage technology with excellent scalability properties. It is used by the [adaptive](/momentum/4/modules/4-adaptive) module, Adaptive Delivery reporting, and Mobile Momentum for resubmission of messages.
 
-Riak is freely available for use in a LAN deployment, but WAN deployments that need to be synchronized require a commercially licensed variant. Note that multiple data centers may still use Riak if each data center has its own local Riak cluster. For more information, see the [Riak](http://http://docs.basho.com/riak/latest/) documentation.
+Riak is freely available for use in a LAN deployment, but WAN deployments that need to be synchronized require a commercially licensed variant. Note that multiple data centers may still use Riak if each data center has its own local Riak cluster. For more information, see the [Riak](http://docs.basho.com/riak/latest/) documentation.
```
Comment (maxharrisnet) [inline content/momentum/4/riak.md:11]: HTTP/S

**Comment by maxharrisnet** [comment]
We can change the urls from`http` to `https`. The rest looks good!

File: components/markdown/link.tsx [hunk 4]
```diff
@@ -27,9 +27,7 @@ export type LinkProps = React.ComponentPropsWithoutRef<'a'> & {
 const AnchorLink = (props: LinkProps): JSX.Element => {
   return (
     <Link passHref href={props.href || '/'}>
-      <StyledLink as="a" href={props.href}>
-        {props.children}
-      </StyledLink>
+      <StyledLink as="a">{props.children}</StyledLink>
```
Comment (nstanard) [inline components/markdown/link.tsx:30]: Wouldn't this result in a console error about href missing from an <a /> tag?

Comment (EvanYoung-SP) [reply components/markdown/link.tsx:30]: So, looking at the documentation here (https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag), if `passHref` is set, it sends the child `a` tag the href, so from my understanding it's doing "double duty" by also setting it in the code on the child `a` tag.

File: content/momentum/4/operations-riak-backups.md (hunk 1 above)
Comment (EvanYoung-SP) [reply content/momentum/4/operations-riak-backups.md:11]: good call. I was just making it not break, but if I'm changing it, might as well update to https.

File: components/markdown/link.tsx (hunk 4 above)
Comment (nstanard) [reply components/markdown/link.tsx:30]: That is a really interesting component integration API.

**Review by maxharrisnet — APPROVED** [review]

**Comment by maxharrisnet** [comment]
Good call on changing the redirects to their final urls. Looks good to me ✅

**Pull request merged**